### PR TITLE
LPS 126220 | Master

### PR DESCRIPTION
**Pull request by CalumR23**
[LPS-126220](https://issues.liferay.com/browse/LPS-126220)

### CODE REVIEW

**Comment by CalumR23** [comment]
ci:test:sf

**Comment by CalumR23** [comment]
ci:test:relevant

**Comment by nikki-pru** [comment]
@CalumR23 Please fix SF errors and resend. Thanks

**Comment by CalumR23** [comment]
ci:test:sf

**Comment by CalumR23** [comment]
ci:test:relevant

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedFieldsetsWebContent.testcase [hunk 1]
```diff
@@ -64,8 +64,49 @@ definition {
...
+		WebContentNavigator.gotoAddWithStructureCP(structureName = "Structure Name Test");
+
+		PortletEntry.inputTitle(title = "WC Title Test");
+
+		PortletEntry.inputText(
+			fieldLabel = "Text",
+			text = "Text test");
+
+		PortletEntry.changeLocaleSummary(locale = "pt_BR");
+
```
Comment (nikki-pru) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedFieldsetsWebContent.testcase:104]: Needs an assert after this to check if the language changed.

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedFieldsetsWebContent.testcase [hunk 2]
```diff
@@ -64,8 +64,49 @@ definition {
...
+		Pause(locator1 = "5000");
+
+		PortletEntry.changeLocale(locale = "es_ES");
+
```
Comment (nikki-pru) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedFieldsetsWebContent.testcase:108]: Also needs an assert here to check that the language changed.

**Review by nikki-pru — CHANGES_REQUESTED** [review]
Please see comments

**Comment by CalumR23** [comment]
ci:test:sf

**Comment by CalumR23** [comment]
ci:test:relevant

**Comment by CalumR23** [comment]
ci:test:sf

**Comment by CalumR23** [comment]
ci:test:relevant

**Comment by nikki-pru** [comment]
ci:stop

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedFieldsetsWebContent.testcase (hunk 1 above)
Comment (CalumR23) [reply modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedFieldsetsWebContent.testcase:106]: Line 107 checks to see if the locale has switched to BR

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedFieldsetsWebContent.testcase (hunk 2 above)
Comment (CalumR23) [reply modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedFieldsetsWebContent.testcase:112]: Line 113 checks to see if the locale has switched to ES

**Comment by nikki-pru** [comment]
See #63

**Pull request closed without merging**